src/index.jsx

```diff
@@ -509,6 +510,12 @@ async function init() {
                     path='/not-found'
                     component={NotFound}
                   />
+                  { userAccessToSite?.enabled && (
```

Comment:
Might add this to all other portal bundles? Or add something in doc saying this feature is available for Gen3 bundle only

Reply:
ill add to the other index pages